public/app/features/dashboard/components/PubdashFooter/index.ts

```diff
@@ -0,0 +1,2 @@
+import PubdashFooter from './PubdashFooter';
```

Comment:
should be able to add the export at the bottom of `PubdashFooter.tsx` like `export default PubdashFooter`

Reply:
actually, it looks like it was already exported. What was this file for?

Reply:
it lets you omit the actual file name in the import (just uses the directory and uses index.ts)

Reply:
I don't see the benefit of this 🤔  What's wrong with having the file name in the import?

`import PubdashFooter from '../components/PubdashFooter/PubdashFooter';`

Reply:
Just saves you some typing I guess. I saw this pattern used in other places in our repo (including several in the same components folder) and preferred it.

Reply:
But yeah, no real benefit lol

Reply:
Normally we use default exports (export default Component) when we connect them to Redux, otherwise it's named export (export const Component), however I don't think this is standardised.

Reply:
Ah okay, I will change it then in this case.